DMTN-049.tex

```diff
@@ -275,117 +276,126 @@ \subsection{Scientific Performance}
-\textit{If possible, describe the minimum set of PZ outputs that are required for your science, or the outputs generated by the PZ estimator(s) you are recommending.
-E.g., full posteriors, point estimates, statistics (mode, mean, standard deviation, skewness, kurtosis), best-fit templates, and/or flags (e.g., quality, failure modes).
-}
-\item Performance -- 
-Rubin DM seeks a general understanding of the minimum quality of PZ estimates needed to meet the basic PZ-related science goals of the community.
-\textit{If possible, describe the minimum PZ quality that would enable your LSST science (e.g., the minimum point-estimate error at intermediate redshifts, or whatever is relevant to your science goals), or the predicted minimum quality of the PZ estimator(s) you are recommending.
-It is understood that this information might not be available at this time.
-}
-\item Technical Aspects -- 
-Rubin DM has a set of technical considerations for PZ estimators, regarding their scalability, inputs, outputs, language, external data sets, training, storage, and compute resources.
-\textit{If possible -- and this is probably only possible for letters being prepared by PZ algorithm developers -- please briefly address the technical considerations described in Section 3 of the PZ Roadmap (ls.st/dmtn-049).
-Details are not necessary, but an evaluation of the technical considerations at the level of ``will meet", ``will probably meet", ``probably will not meet", ``will not meet" would be most helpful at this time.
-}
-\end{enumerate}
 
-\subsection{Shortlisted Photo-$z$ Estimator(s)} \label{ssec:lor_choice}
+\subsection{Seeking Photo-$z$ Expertise for the Rubin Commissioning Team}\label{ssec:pzcoop_ct}
 
-\textit{TBD: This section will summarize the "Letters of Recommendation" and describe which community-vetted photo-$z$ estimator(s) have been shortlisted for the photo-$z$ validation cooperative, and why.}
+It is expected that \textit{most} of the science community participating in the ``Photo-$z$ Validation Cooperative" will use the commissioning data that is released as part of Data Preview 1 and 2 (DP1 and DP2), and will not need to formally join the Rubin Commissioning Team and have access to the unreleased commissioning data.
 
-\section{Photo-$z$ Validation Cooperative}\label{sec:pzcoop}
+However, as mentioned above the DM team is very limited in terms of the support it can provide for implementation, and furthermore there are some aspects of implementing and validating photo-$z$ estimators which would benefit from a connection with the unreleased commissioning data.
 
-\textbf{This preliminary draft description of this proposed cooperative effort will be updated based on community input.}
+Thus, Rubin Observatory is seeking to identify a small group of $\sim$5-10 individuals who have expertise with photo-$z$ estimators to join the Rubin Commissioning Team.
```

Comment:
Re. downselecting, my impression is that a team of 5-10 is appropriate and matches other Commissioning AO teams, and yes we should downselect if this opportunity is oversubscribed. The "PZ-related" Commissioning team members should be limited to only people who are committing time and effort to implementation, and their statements must be reviewed like the other Commissioning AO teams. Also, the 5-10 number is probably appropriate because DM is limited in how many people it can help with PZ implementation. All this comes from talking with Keith and Jim about how these PZ-related Commissioning Team members would work. 

Re. "anyone can participate", later on we do say that anyone interested in committing to Commissioning Team participation can reach out (to me) to put together a statement. This is not limited to PZ LOR writers nor the shortlisted estimator teams.

Reply:
OK, That all makes sense and I agree with all the points.